src/drive/web/modules/actions/index.jsx

```diff
@@ -80,9 +80,7 @@ export const download = ({ client, vaultClient }) => {
           const { t } = useI18n()
           return (
             <ActionMenuItem
-              onClick={() => {
-                return exportFilesNative(client, props.files, { vaultClient })
-              }}
+              onClick={props.onClick}
```

Comment:
Where does this onClick props come from?

Reply:
Does it come from https://github.com/cozy/cozy-drive/blob/a731531b6d2077fd82e5edb3b751bd41530bb120/src/drive/web/modules/actionmenu/ActionsItems.jsx#L79 ? 

ActionsItems are used for the selection bar but also for the "..." ?

Reply:
> Does it come from ... ?

Yep that's it:
```js
    const onClick = action
      ? () => {
          action([file])
          onClose()
        }
      : null```
https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/actionmenu/ActionsItems.jsx#L62.

You can check the `onClick` I remove is the exact duplicate of the `action` fields setup in the same object. So instead of duplicating you can use this `onClick` function which was already given in props.